### PR TITLE
Kf 2316 integration tests for sklearnserver

### PR DESCRIPTION
**Pull request by i-chvets**
Summary of changes:
    - Added test YAMLs for sklearn server.
    - Implemented sklearn server integration tests.

Tests are passing locaaly:

```
$ tox -e integration
integration: install_deps> python -I -m pip install -r requirements-integration.txt
integration: commands[0]> pytest -v --tb native --asyncio-mode=auto /home/ichvets/cw/dev/seldon-core-operator/tests/integration --log-cli-level=INFO -s
=========================================================== test session starts ============================================================
platform linux -- Python 3.8.16, pytest-7.3.1, pluggy-1.0.0 -- /home/ichvets/cw/dev/seldon-core-operator/.tox/integration/bin/python
cachedir: .tox/integration/.pytest_cache
rootdir: /home/ichvets/cw/dev/seldon-core-operator
configfile: pyproject.toml
plugins: asyncio-0.21.0, operator-0.27.0, anyio-3.6.2
asyncio: mode=auto
collected 8 items                                                                                                                          

tests/integration/test_charm.py::test_build_and_deploy 
-------------------------------------------------------------- live log setup --------------------------------------------------------------
INFO     pytest_operator.plugin:plugin.py:647 Adding model uk8s:test-charm-gojm on cloud microk8s
-------------------------------------------------------------- live log call ---------------------------------------------------------------
INFO     pytest_operator.plugin:plugin.py:526 Using tmp_path: /home/ichvets/cw/dev/seldon-core-operator/.tox/integration/tmp/pytest/test-charm-gojm0
INFO     pytest_operator.plugin:plugin.py:978 Building charm seldon-core
INFO     pytest_operator.plugin:plugin.py:983 Built charm seldon-core in 28.58s
INFO     juju.model:model.py:1960 Deploying local:seldon-core-0
INFO     juju.model:model.py:2584 Waiting for model:
  seldon-controller-manager/0 [allocating] waiting: installing agent
INFO     juju.model:model.py:2584 Waiting for model:
  seldon-controller-manager/0 [allocating] waiting: agent initializing
INFO     juju.model:model.py:2584 Waiting for model:
  seldon-controller-manager/0 [idle] active: 
PASSED
tests/integration/test_charm.py::test_seldon_istio_relation 
-------------------------------------------------------------- live log call ---------------------------------------------------------------
INFO     juju.model:model.py:1960 Deploying ch:amd64/focal/istio-gateway-526
INFO     juju.model:model.py:1960 Deploying ch:amd64/focal/istio-pilot-527
INFO     juju.model:model.py:2584 Waiting for model:
  istio-pilot/0 [allocating] waiting: installing agent
  istio-ingressgateway/0 [allocating] waiting: agent initializing
INFO     juju.model:model.py:2584 Waiting for model:
  istio-pilot/0 [allocating] waiting: agent initializing
  istio-ingressgateway/0 [idle] waiting: List of <ops.model.Relation istio-pilot:0> versions not found for apps: istio-pilot
INFO     juju.model:model.py:2584 Waiting for model:
  istio-pilot/0 [idle] active: 
  istio-ingressgateway/0 [idle] active: 
INFO     juju.model:model.py:2584 Waiting for model:
  seldon-controller-manager/0 [idle] active: 
  istio-ingressgateway/0 [idle] active: 
  istio-pilot/0 [executing] active: 
PASSED
tests/integration/test_charm.py::test_seldon_alert_rules 
-------------------------------------------------------------- live log call ---------------------------------------------------------------
INFO     juju.model:model.py:1960 Deploying ch:amd64/focal/prometheus-k8s-103
INFO     juju.model:model.py:2584 Waiting for model:
  prometheus-k8s/0 [allocating] waiting: installing agent
INFO     juju.model:model.py:2584 Waiting for model:
  prometheus-k8s/0 [executing] waiting: Waiting for resource limit patch to apply
INFO     juju.model:model.py:2584 Waiting for model:
  prometheus-k8s/0 [idle] waiting: Waiting for resource limit patch to apply
INFO     juju.model:model.py:2584 Waiting for model:
  prometheus-k8s/0 [executing] maintenance: 
INFO     juju.model:model.py:2584 Waiting for model:
  prometheus-k8s/0 [idle] active: 
INFO     httpx:_client.py:1013 HTTP Request: POST https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments "HTTP/1.1 201 Created"
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == None (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model-1 status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:97 seldondeployment/seldon-model-1 status == Available
INFO     httpx:_client.py:1013 HTTP Request: DELETE https://10.1.1.124:16443/apis/apps/v1/namespaces/test-charm-gojm/deployments/seldon-model-1-example-0-classifier "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:124 Waiting for alert state to propagate to Prometheus
INFO     test_charm:test_charm.py:124 Waiting for alert state to propagate to Prometheus
INFO     test_charm:test_charm.py:124 Waiting for alert state to propagate to Prometheus
INFO     httpx:_client.py:1013 HTTP Request: DELETE https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model-1 "HTTP/1.1 200 OK"
PASSED
tests/integration/test_charm.py::test_seldon_deployment 
-------------------------------------------------------------- live log call ---------------------------------------------------------------
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: PATCH https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: POST https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments "HTTP/1.1 201 Created"
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model status == None (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/seldon-model status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/seldon-model "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:97 seldondeployment/seldon-model status == Available
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm/services/seldon-model-example-classifier "HTTP/1.1 200 OK"
PASSED
tests/integration/test_charm.py::test_seldon_predictor_server[sklearn.yaml-api/v1.0/predictions-req_data0-resp_data0] 
-------------------------------------------------------------- live log call ---------------------------------------------------------------
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: PATCH https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: POST https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments "HTTP/1.1 201 Created"
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == None (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:97 seldondeployment/sklearn status == Available
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm/services/sklearn-default-classifier "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: DELETE https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: DELETE https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm/services/sklearn-default-classifier "HTTP/1.1 200 OK"
PASSED
tests/integration/test_charm.py::test_seldon_predictor_server[sklearn-v2.yaml-v2/models/classifier/infer-req_data1-resp_data1] 
-------------------------------------------------------------- live log call ---------------------------------------------------------------
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: PATCH https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: POST https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments "HTTP/1.1 201 Created"
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == None (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:99 seldondeployment/sklearn status == Creating (waiting for 'Available')
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     test_charm:test_charm.py:97 seldondeployment/sklearn status == Available
INFO     httpx:_client.py:1013 HTTP Request: GET https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm/services/sklearn-default-classifier "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: DELETE https://10.1.1.124:16443/apis/machinelearning.seldon.io/v1/namespaces/test-charm-gojm/seldondeployments/sklearn "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1013 HTTP Request: DELETE https://10.1.1.124:16443/api/v1/namespaces/test-charm-gojm/services/sklearn-default-classifier "HTTP/1.1 200 OK"
PASSED
```

### CODE REVIEW

**Comment by i-chvets** [comment]
> name of the PR sounds confusing. This is "integration tests for ....."
Not sure what you mean?
PR is to integrate sklearnsever rock. Integration tests are for sklearn server.
> - Implemented sklearn server integration tests.

File: tests/integration/test_charm.py [hunk 1]
```diff
@@ -281,6 +281,103 @@ async def test_seldon_deployment(ops_test: OpsTest):
...
+                        "data": [2],
+                    }
+                ],
+            },
+        ),
+    ],
+)
+@pytest.mark.asyncio
+async def test_seldon_server(ops_test: OpsTest, server_config, url, req_data, resp_data):
+    """Test Seldon server scenario."""
```
Comment (beliaev-maksim) [inline tests/integration/test_charm.py:336]: can we be more specific on what is tested?

File: examples/sklearn-v2.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - graph:
+      children: []
+      implementation: SKLEARN_SERVER
+      modelUri: gs://seldon-models/v1.17.0-dev/sklearn/iris
+      name: classifier
+      parameters:
+        - name: method
+          type: STRING
+          value: predict
+    name: default
```
Comment (beliaev-maksim) [inline examples/sklearn-v2.yaml:19]:
```suggestion
    name: default

```

File: tests/integration/test_charm.py (hunk 1 above)
Comment (i-chvets) [reply tests/integration/test_charm.py:336]: There are multiple Seldon servers defined in [configmap](https://github.com/canonical/seldon-core-operator/blob/main/src/templates/configmap.yaml.j2) under `predictor_servers`. I will rename it to that.

**Review by beliaev-maksim — APPROVED** [review]

**Pull request merged**